models/snkrsDao.js

```diff
@@ -0,0 +1,221 @@
+import { PrismaClient } from '@prisma/client';
+
+const prisma = new PrismaClient();
+
+const updataOpenClose = async (bool, style_code) => {
+  return await prisma.$queryRaw`
+    UPDATE
+      snkrs
+    SET
+      is_open = ${bool}
+    WHERE
+      style_code = ${style_code}
+  `;
+};
+
+const addLottoBox = async (user_id, style_code, size) => {
+  return await prisma.$queryRaw`
+    INSERT INTO
+      snkrs_data(
+        style_code,
+        user_id,
+        size,
+        is_winner
+      )
+    VALUES(
+      ${style_code},
+      ${user_id},
+      ${size},
+      false
+    );
+  `;
+};
+
+const existLottoBox = async (user_id, style_code) => {
+  return await prisma.$queryRaw`
+    SELECT EXISTS(
+      SELECT
+        style_code,
+        user_id
+      FROM
+        snkrs_data
+      WHERE
+        user_id = ${user_id} AND style_code = ${style_code}
+    ) as result
+    `;
+};
+
+const existWinnerBox = async style_code => {
+  return await prisma.$queryRaw`
+    SELECT EXISTS(
+      SELECT
+        style_code,
+        user_id
+      FROM
+        snkrs_winners
+      WHERE
+        style_code = ${style_code}
+    ) as result;
+    `;
+};
+
+const getNumOfParticipants = async style_code => {
+  return await prisma.$queryRaw`
+    SELECT
+      style_code,
+      user_id
+    FROM
+      snkrs_data
+    WHERE
+      style_code = ${style_code};
+  `;
+};
+
+const getCount = async style_code => {
+  return await prisma.$queryRaw`
+    SELECT
+      MAX(count)
+    FROM  
+      snkrs_winners
+    WHERE
+      snkrs_winners.style_code = ${style_code};
+  `;
+};
+
+const selectWinner = async style_code => {
+  return await prisma.$queryRaw`
+    SELECT
+      style_code,
+      user_id,
+      size
+    FROM
+      snkrs_data
+    WHERE
+      style_code = ${style_code}
+    ORDER BY
+      rand()
+    LIMIT
+      1; 
+  `;
+};
+
+const addWinnerBox = async (style_code, user_id, size, is_winner, count) => {
+  return await prisma.$queryRaw`
+    INSERT INTO
+      snkrs_winners(
+        style_code,
+        user_id,
+        size,
+        is_winner,
+        count
+      )
+    VALUES(
+      ${style_code},
+      ${user_id},
+      ${size},
+      ${is_winner},
+      ${count}
+    );
+  `;
+};
+
+const deleteLottoBox = async style_code => {
+  return await prisma.$queryRaw`
+    DELETE FROM
+      snkrs_data
+    WHERE
+      style_code = ${style_code};
+  `;
+};
+
+const updateWinner = async (style_code, user_id) => {
+  return await prisma.$queryRaw`
+    UPDATE
+      snkrs_winners
+    SET
+      is_winner = true
+    WHERE
+      style_code = ${style_code} AND user_id = ${user_id}
+    ORDER BY
+      create_at DESC
+    LIMIT
+      1;
+  `;
+};
```

Comment:
true를 parameter로 받아올 수 있으면 좋겠네요 :)
```suggestion
const updateWinner = async (style_code, user_id) => {
  return await prisma.$queryRaw`
    UPDATE
      snkrs_winners
    SET
      is_winner = ${___}
    WHERE
      style_code = ${style_code} AND user_id = ${user_id}
    ORDER BY
      create_at DESC
    LIMIT
      1;
  `;
};
```